google/cloud/apigee/registry/applications/v1alpha1/diff_analytics.proto

```diff
@@ -43,3 +43,15 @@ message ClassifiedChanges {
   Diff unknownChanges = 3;
 }
 
+message Stats {
+  int64 totalChanges = 1;
```

Comment:
These field names should be in snake_case instead of camelCase. I didn't notice this on the ClassifiedChanges message, but that can be corrected in a different PR.